website/skirolly/index.qmd

```diff
@@ -17,59 +17,66 @@ format:
 title-block-banner: false
 ---
 
-:::{.cr-section}
-Skiers and riders, you’ve all seen these trail maps—they're essential for navigating the mountain, finding the fastest lift, or planning your next epic run. @cr-southface
+::::{.cr-section}
+
+Skiers and riders, you’ve all seen these trail maps —
+they're essential for navigating the mountain, finding the fastest lift, or planning your next run.
+@cr-southface
 
-In particular, this map, beautifully painted by the legendary [James Niehues](https://en.wikipedia.org/wiki/James_Niehues), is of the <span style="color: #FF8C42;">**South Face**</span> of Big Sky, Montana.
+This map,
+painted by the legendary [James Niehues](https://en.wikipedia.org/wiki/James_Niehues),
+shows the [**South Face**]{style="color: #FF8C42;"} of Big Sky, Montana, 🇺🇸.
 
-But have you ever wondered how common south-facing slopes are? In other words, do most ski trails in this hemisphere favor <span style="color: #FF8C42;">**southern**</span> exposure, or are they more often found on the <span style="color: #36B37E;">**north**</span> side of the mountain?
+But have you ever wondered how common south-facing slopes are?
+In other words, do most ski trails in this hemisphere favor [**southern**]{style="color: #FF8C42;"} aspects,
+or are they more often found on the [**north**]{style="color: #36B37E;"} side of the mountain?
 @cr-southface
 
 :::{#cr-southface}
 ![](https://www.myskimaps.com/Ski-Maps/USA/Big-Sky-South-Face-Ski-Trail-Map-2019.jpg)
 :::
-:::
+::::
 
 ::::{.cr-section layout="overlay-center"}
-We began our journey to answer this question by gathering data from [OpenSkiMap](https://openskimap.org/), which refines [OpenStreetMap](https://www.openstreetmap.org/) 
-to create an amazing interactive visualization of the world and all its ski trails. 
+We began our journey to answer this question by gathering data from [OpenSkiMap](https://openskimap.org/),
+which refines [OpenStreetMap](https://www.openstreetmap.org/) to create an amazing interactive visualization of the world and all its ski trails.
 @cr-openskimap
 
 :::{#cr-openskimap}
 ![](https://github.com/dhimmel/openskistats/raw/assets/assets/openskimap.jpg){style="border-radius: 0%;"}
 :::
 
 Thousands of OpenStreetMap contributors have worked for decades to annotate ski areas and their trails.<br><br>
-In this New Hampshire/Vermont region, ...
+In the New Hampshire/Vermont region, ...
```

Comment:
by "this" I wanted to point to this particular view that we're showing of the map. I also debated "Upper Valley" but maybe that would alienate many readers.